src/shared/src/components/LandmarkShortcuts.tsx

```diff
@@ -0,0 +1,65 @@
+import { makeStyles } from "@material-ui/core";
+import React from "react";
+import cn from 'classnames';
+
+export interface ILandmarkShortcutsProps {
+  navToMainText: string;
+}
+
+const useStyles = makeStyles({
+  button: {
+    textDecoration: 'underline',
+    textAlign: 'left',
+    width: 'fit-content',
+  },
+  jumpNav: {
+    backgroundColor: "#fff",
+  },
+  'sr-only': {
```

Comment:
Given this likely not being the only place where sr-only needs to be used, maybe make it an export in a `srOnly.ts` file and use spreading where it is applicable?
```js
import {srOnly} from ´altinn-shared/styles´;
...
'sr-only': {
{...srOnly}
...
}
```